Formula/prestodb.rb

```diff
@@ -63,11 +64,11 @@ def install
     (libexec/"etc/catalog/jmx.properties").write "connector.name=jmx"
 
     rewrite_shebang detected_python_shebang, libexec/"bin/launcher.py"
-    (bin/"presto-server").write_env_script libexec/"bin/launcher", Language::Java.overridable_java_home_env
+    (bin/"presto-server").write_env_script libexec/"bin/launcher", JAVA_HOME: Formula["openjdk@11"].opt_prefix
```

Comment:
Why not?

```suggestion
    (bin/"presto-server").write_env_script libexec/"bin/launcher", Language::Java.overridable_java_home_env("11")
```

Reply:
Pushed the change, but not sure how to how to tell CI it fails on ARM.

```
Presto requires x86_64 on Mac OS X (found aarch64)
```

Reply:
Looks like this formula never worked for Apple Silicon, so I guess we can add `depends_on arch: :x86_64` (with a reference to https://github.com/prestodb/presto/issues/17146) just to get bottles built.